template.yaml

```diff
@@ -572,6 +592,31 @@ Resources:
                 -  "s3:GetObjectVersion"
               Resource:
                 - !Join ['/', [!GetAtt BatchJobManifestBucket.Arn, '*']]
+      - PolicyName: AllowS3Copy
+        PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+            - Effect: Allow
```

Comment:
Can we put some Sid entries on these policies so it's a bit clearer what they're for?

i.e. AllowCopyToAnalysisBucket.

Or maybe they should be separate policies?

Basically one policy is for the audit bucket, and another for the analysis bucket which is a tad confusing to me!

Reply:
I've added Sids to everything now, is that better? I'm reluctant to pull it out any more, unless you think there's a good reason for it?